test/sablon_test.rb

```diff
@@ -187,3 +187,25 @@ def test_generate_document_from_template
     assert_equal "Filename: 'clone' has no discernable extension", e.message
   end
 end
+
+class SablonImagesTest < Sablon::TestCase
```

Comment:
Can you change this to `SablonSvgImageTest`? That way the name doesn't conflict with the above class.

Reply:
Copy-paste strikes again. Fixed.